pkg/common/configs/configvalidator.go

```diff
@@ -539,8 +539,8 @@ func checkLimit(limit Limit, existedUserName map[string]bool, existedGroupName m
 				zap.Error(err))
 			return err
 		}
-		if resources.IsZero(limitResource) {
-			return fmt.Errorf("MaxResources is zero in '%s' limit, all resource types are zero", limit.Limit)
+		if !resources.StrictlyGreaterThanZero(limitResource) {
+			return fmt.Errorf("MaxResources is not greater than zero in '%s' limit", limit.Limit)
```

Comment:
MaxResources is not greater than zero in '%s' limit -> MaxResources should be greater than zero in '%s' limit ?

Reply:
Updated it. Thank you.